bundles/org.openhab.binding.amazonechocontrol/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="amazonechocontrol:smartHomeDevice">
+		<instruction-set targetVersion="1">
+			<update-channel id="colorTemperatureInKelvin">
+				<type>system:color-temperature-abs</type>
```

Comment:
Checked documentation and 10+ instructions.xml files but i don't get it, could you elaborate on on your comment?

Reply:
I don't find an example but I am sure this case was already encountered and discussed.
I will have to search in the archives.
Maybe the solution was to remove the channel and to add it again.

Reply:
I'll wait to merge before you have something. Maybe if you know who was envolved you can ping them to join?

Reply:
@jlaur : do you remember?

Reply:
@J-N-K do you also remember?

Reply:
I don't remember if this will work, and it might take two minutes (with a warning logged) when the channel type for an existing Thing is completely missing during startup (because it's deleted). It might be a variant of one of the issues mentioned here: openhab/openhab-core#3660

Reply:
Tested this (see https://github.com/openhab/openhab-addons/pull/17754#issuecomment-2495681675 ) No need to add additional attributes to the ugprade instructions.